packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -327,13 +334,33 @@ abstract class BindingBase {
     registerServiceExtension(
       name: name,
       callback: (Map<String, String> parameters) async {
-        if (parameters.containsKey(name))
-          await setter(double.parse(parameters[name]));
+        if (parameters.containsKey(name)) {
+          final double value = double.parse(parameters[name]);
+          await setter(value);
+          _postExtensionToggledEvent(name, value);
+        }
         return <String, dynamic>{ name: (await getter()).toString() };
       }
     );
   }
 
+  void _postExtensionToggledEvent(String name, dynamic value) {
+    postEvent(
+      'Flutter.ServiceExtensionToggled',
+      <String, dynamic>{
+        'extension': name,
+        'value': value,
+      },
+    );
+  }
+
+  /// Use this method instead of calling [developer.postEvent] directly so that
```

Comment:
make this comment exactly match the style of other comments in package:flutter indicating the same thing. If the other comments aren
t consistent, bonus points for making them consistent.
For example:
```
  /// All events dispatched by a [WidgetInspectorService] use this method
  /// instead of calling [developer.postEvent] directly so that tests for
  /// [WidgetInspectorService] can track which events were dispatched by
  /// overriding this method.
```

Reply:
done.